### PR TITLE
wuzz 0.4.0 (new formula)

### PR DESCRIPTION
**Pull request by YuerLee**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/wuzz.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
+require "language/go"
+
+class Wuzz < Formula
+  desc "Interactive cli tool for HTTP inspection"
```
Comment (bfontaine) [inline Formula/wuzz.rb]: you might want to write either "`CLI`" or "`command-line`" for consistency.

File: Formula/wuzz.rb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  homepage "https://github.com/asciimoo/wuzz"
+  url "https://github.com/asciimoo/wuzz.git",
+      :tag => "v0.4.0",
+      :revision => "ef041bc912ea39081ddc5f8b3896ded90e8b053b"
+  head "https://github.com/asciimoo/wuzz.git",
+       :shallow => false
```
Comment (bfontaine) [inline Formula/wuzz.rb]: Why `:shallow => false`?

**Review by bfontaine — CHANGES_REQUESTED** [review]

File: Formula/wuzz.rb (hunk 2 above)
Comment (passerbyid) [reply Formula/wuzz.rb]: It doesn't need to be false, I will fix it.

File: Formula/wuzz.rb (hunk 1 above)
Comment (passerbyid) [reply Formula/wuzz.rb]: I just copy repo description of wuzz, I will change it to `CLI` for consistency.

**Review by ilovezfs — CHANGES_REQUESTED** [review]
In new formulae, we want to avoid `go_resource` blocks. Please file an issue upstream requesting use of a Go vendor directory or one of the various Go vendoring tools.

File: Formula/wuzz.rb [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p "src/github.com/asciimoo"
+    ln_s buildpath, "src/github.com/asciimoo/wuzz"
+    Language::Go.stage_deps resources, buildpath/"src"
+    system "go", "build", "-o", bin/"wuzz"
+  end
+
+  test do
+    system bin/"wuzz", "-h"
```
Comment (ilovezfs) [inline Formula/wuzz.rb]: The test needs to exercise and verify the functionality of the software.

**Comment by ilovezfs** [comment]
Please feel free to reopen when the above issues have been addressed!

**Pull request closed without merging**